packages/expect/src/utils.js

```diff
@@ -114,16 +114,29 @@ export const iterableEquality = (a: any, b: any) => {
   if (a.size !== undefined) {
     if (a.size !== b.size) {
       return false;
-    } else {
-      const args = [];
+    } else if (a instanceof Set) {
```

Comment:
With this change, this branch will pass if b is a superset of a, which I don't think is intentional.

Reply:
I don't see how that follows. `b` must contain all the elements of `a`, and be the same size. The size constraint should preclude a superset from matching (and there is a test for that).

Reply:
Good point, I didn't realize that we had this check up there.